docs/api.apib

```diff
@@ -88,6 +113,52 @@ Creates a new image based on a JSON object containing the following metadata.
                 "required": [ "label","published_at", "url" ]
             }
 
+All fields are required, 
```

Comment:
This actually produces a weird output on docs:

![image](https://user-images.githubusercontent.com/25080094/36602708-78034e38-1897-11e8-94a1-164e0e7a3dfe.png)

Reply:
With markdown rendering this isn't an issue, I'll fix it